apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -148,10 +150,19 @@ export default {
       required: false,
       default: () => null,
     },
+    locale: {
+      type: String,
+      default: () => "en",
```

Comment:
Is there a preference of using an ()=> 'defaultValue' over just directly saying 'defaultValue'? We are using both in our application, but I don't really know the